### PR TITLE
#123 integrate cloudfalre turnstile re captcha

### PR DESCRIPTION
**Pull request by Mikivitkovski**

### CODE REVIEW

**Review by edichoska — COMMENTED** [review]
@Mikivitkovski we should be sending the token from the captcha to the backend.

**Comment by Mikivitkovski** [comment]
> @Mikivitkovski we should be sending the token from the captcha to the backend.

Please check i think is resolved now

File: components/reusable-components/newsletter/Newsletter.tsx [hunk 1]
```diff
@@ -1,6 +1,13 @@
+'use client';
+
 import React from 'react';
 import styles from './newsletter.module.scss';
 import EmailForm from './emailform/emailform';
+import CaptchaWidget from '../turnstile-captcha/CaptchaWidget';
+
+const handleCaptchaSuccess = () => {};
```
Comment (stoimenov95) [inline components/reusable-components/newsletter/Newsletter.tsx:8]: @Mikivitkovski why are these empty functions?

**Review by stoimenov95 — CHANGES_REQUESTED** [review]
Screenshot 2024-04-16 at 22 20 17
@Mikivitkovski please build the code and make sure we don't have any errors, then make another request for a review.

**Review by edichoska — APPROVED** [review]

**Review by stoimenov95 — APPROVED** [review]

**Pull request merged**